yarn.lock

```diff
@@ -18,7 +18,7 @@
   optionalDependencies:
     chokidar "^2.1.8"
 
-"@babel/code-frame@7.10.4":
+"@babel/code-frame@7.10.4", "@babel/code-frame@^7.10.4":
```

Comment:
Could you please revert the changes to `yarn.lock`, then re-run `yarn install`? See https://beyondessential.slack.com/archives/CDGS6P56K/p1633403340044000 for more

Reply:
Woops I thought I did that but must have cleared it with my many yarn installs. I have reset and re built it and it's looking good now.